bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -152,6 +154,21 @@ For example:
 | 0xB4CA    | 8        | 0xB4CA8  |
 | 0xB4CA    | 0        | 0xB4CA0  |
 
+## Using the group 0
```

Comment:
```suggestion
## Using Group 0
```
Headings have each word capitalised. If you accept the change, be sure to add the signoff as the last line.